spark/dl/src/main/scala/com/intel/analytics/bigdl/transform/vision/image/MTImageFeatureToBatch.scala

```diff
@@ -269,8 +270,8 @@ object RoiImageInfo {
  * RoiImageInfo.IMGINFO  with shape (batchSize, 4), contains all images info
  *                 (height, width, original height, original width)
  */
-class RoiMiniBatch(val input: Tensor[Float], val target: Array[RoiLabel],
-  val isCrowd: Array[Tensor[Float]], val imageInfo: Tensor[Float] = null)
+class RoiMiniBatch(val input: Tensor[Float], val target: SerializableIndexedSeq[RoiLabel],
+  val isCrowd: SerializableIndexedSeq[Tensor[Float]], val imageInfo: Tensor[Float] = null)
```

Comment:
Array by default support serialization, right ?

Reply:
But the `slice` method of RoiMiniBatch uses array.slice, may not be efficient.
One of the old commit uses IndexSeq and array.view to achieve better performance but it causes serialization problems.